### PR TITLE
Load Pocket GA only after OneTrust active signal (Fixes #12670)

### LINKED ISSUE #12670: Pocket site should only load GA after it gets an active signal from OneTrust
## Description

The logged out Pocket marketing pages should only be loading GA after we get a signal from the OneTrust cookie consent banner.

### Steps to reproduce

1. Open https://getpocket.com/en/about/
2. Click "Manage" in the cookie consent popup
3. Click "Analytics Cookies" in the cookie preferences window.
4. Click the toggle to disable analytics cookies.
5. Click "Confirm"
6. Reload the page.

### Expected result

Google Analytics should not be loaded.

### Actual result

Google Analytics is still loaded.

### Environment

Reproducible in any browser

### PR DESCRIPTION
**Pull request by alexgibson**
## One-line summary

Adds a check to the Pocket site to only load GA after we get an active signal from the OneTrust cookie consent banner.

## Significant changes and points to review

In order for the cookie consent banners to work on non-production domains, we need to load a different test script when `Dev=True`. This should help us QA changes more easily in the future.

## Issue / Bugzilla link

#12670

## Testing

Demo link: https://www-demo1.tekcopteg.com/en/

To test locally, you'll need to set the following `.env` variables:

```
GTM_CONTAINER_ID=G-NFR9Y40GD3
GOOGLE_ANALYTICS_ID=UA-370613-9
```

- [x] On initial load, cookie consent banner is shown and both GA scripts are loaded.
- [x] Click "Manage" -> "Analytics Cookies" -> "Disable" -> "Confirm".
- [x] After refreshing the page, cookie consent banner should not be shown and GA scripts should not be loaded.
- [x] Opting back in should load GA as expected.
- [x] Clicking "Accept all" and "Reject all" should also work as expected.

### CODE REVIEW

**Comment by alexgibson** [comment]
Successful test run: https://gitlab.com/mozmeao/www-config/-/pipelines/763946915

**Comment by alexgibson** [comment]
This is ready for review

File: bedrock/settings/__init__.py [hunk 1]
```diff
@@ -102,6 +102,13 @@
     DEV_LANGUAGES = PROD_LANGUAGES
     LANGUAGE_CODE = "en"  # Pocket uses `en` not `en-US`
 
+    COOKIE_CONSENT_SCRIPT_SRC = (
+        "https://cdn.cookielaw.org/scripttemplates/otSDKStub.js"
+        if DEV
+        else "https://cdn.cookielaw.org/consent/a7ff9c31-9f59-421f-9a8e-49b11a3eb24e/otSDKStub.js"
+    )
+    COOKIE_CONSENT_DATA_DOMAIN = "a7ff9c31-9f59-421f-9a8e-49b11a3eb24e-test" if DEV else "a7ff9c31-9f59-421f-9a8e-49b11a3eb24e"
```
Comment (maureenlholland) [inline bedrock/settings/__init__.py:110]: 👍

Comment (maureenlholland) [reply bedrock/settings/__init__.py:110]: Do we need to add these in mozmeao www-config as well? Do we need to notify SRE to update anything the GCP settings? cc: @stevejalim

**Review by maureenlholland — APPROVED** [review]
All working as described r+ ✅ 

Couple of questions around how the new cookie consent settings get deployed across environments, but nothing blocking for Bedrock changes

File: bedrock/settings/__init__.py (hunk 1 above)
Comment (alexgibson) [reply bedrock/settings/__init__.py:110]: These settings only need to change depending on if Dev=True or not, so shouldn’t need any env specific variables set, I think?

**Pull request merged**